src/store/reducers/settings.js

```diff
@@ -33,7 +36,7 @@ export const initialState = {
   discreetMode: false,
   token: {
     active: tokenKeys[0],
-    list: tokenKeys.reduce((acc, key) => { acc[key] = true; return acc; }, {}),
+    list: tokenList,
```

Comment:
As we are only going to support LSK can we hardcode all of this to simply:
```
{
  active: tokenKeys.LSK.key,
  list: {
    tokenKeys.LSK.key: true,
    tokenKeys.BTC.key: false,
}
```
Reducing tokenKeys to generate the list looks fine in the case we have more tokens to support, but given that we only support LSK seems overengineering to me.

Reply:
@isalga Well I left that possibility in the event we want to support another coin in the near future but what you suggested makes sense and I'll update accordingly